org-formation/050-costs/_tasks.yaml

```diff
@@ -20,6 +20,22 @@ AnomalyDetectorService:
   Parameters:
     Subscriber: !GetAtt CurrentAccount.RootEmail
 
+# Deploy a general-use microservice for interacting with MIPS in admincentral
+MipsMicroservice:
+  Type: update-stacks
+  Template: !Sub 'https://${AdminCentralCfnBucket}.s3.amazonaws.com/lambda-mips-api/master/lambda-mips-api.yaml'
+  StackName: !Sub '${resourcePrefix}-mips-microservice'
+  DefaultOrganizationBinding:
+    IncludeMasterAccount: false
+    Account: !Ref AdminCentralAccount
+    Region: !Ref primaryRegion
+  Parameters:
+    MipsOrganization: 'SAGE_24146'
+    SsmKeyAdminArns:
+      - 'arn:aws:sts::745159704268:assumed-role/AWSReservedSSO_Administrator_30244677b3ea9498/joni.harker@sagebase.org'
```

Comment:
i don't think @xschildw nor I need access.  You'll probably also want to give root user and CI role access with this these ARNs:

```
- !Sub 'arn:aws:iam::${CurrentAccount.AccountId}:root'
- 'arn:aws:iam::745159704268:role/github-oidc-sage-bionetwo-ProviderRoleorganization-93H11ERK3F4N'
```

Reply:
done